macros/public/get_chain_explorer_tx_hash.sql

```diff
@@ -19,4 +19,4 @@
          when 'goerli' = chain_ then 'https://goerli.basescan.org/tx/' || CAST(hash_ AS VARCHAR)
```

Comment:
Is this goerli meant to for ethereum or base?

Reply:
not sure, tbh. just kept the original macro